DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/filter.ts

```diff
@@ -112,9 +112,9 @@ function filterElements(
   );
 }
 
-function appendNewFilterToFilterBy(filterSubmit: HTMLInputElement): string {
+function appendNewFilterToExistingFilterString(filterSubmit: HTMLInputElement): string {
   const filterValue = getSelectedFilterFromDropdownAndResetDropdown(filterSubmit);

```

Comment:
newFilterToAdd? Or is this just the actual value and not the full formatted string with filter name etc.?